### DISCUSSION TITLE
Does this Plugin Expose Fan Controls to Homekit?

### DISCUSSION THREAD

**Discussion by Cuelist2000** [d0n4v4nb3ck3r/homebridge-honeywell-home#734] [General]
Hi guys, first of all....amazing job with the plugin!

I have a question, The Honeywell app has fan control (not AC or heat but just the fan) it does Fan On, Fan Only when system is on and a feature called circulate...which turns on the fan randomly for a period of time to just circulate air.  Does anyone know if this plugin exposes that feature to HomeKit?  it would be cool to create some automations based on humidity or air quality in conjunction with a weather plugin!!!

**Comment by donavanbecker** [comment]
Yes it should. Depending on your model.

**Comment by Cuelist2000** [comment]
T6 is the model...will try it out
